### ISSUE TITLE
tap missing src_addr on outbound requests

### ISSUE THREAD

**Issue by olix0r** [linkerd/linkerd2#5091]
We recently stopped exposing source addresses on outbound tap metadata, but this leads to a pretty clunky UX:

```
req id=2:79 proxy=out src=<nil>:0 dst=10.42.0.20:8080 tls=true :method=POST :authority=voting-svc.emojivoto:8080 :path=/emojivoto.v1.VotingService/VoteMassageWoman
rsp id=2:79 proxy=out src=<nil>:0 dst=10.42.0.20:8080 tls=true :status=200 latency=2658µs
end id=2:79 proxy=out src=<nil>:0 dst=10.42.0.20:8080 tls=true grpc-status=OK duration=110µs response-length=5B
```

We should restore this metadata before the stable release.

**Comment by hawkw** [comment]
Did linkerd/linkerd2-proxy#718 fix this, or is there something else I need to be working on?

**Comment by olix0r** [comment]
@hawkw I'll close the issue when the proxy release is merged over here.

**Comment by olix0r** [comment]
Closed by #5102
